hack/test-cmd.sh

```diff
@@ -57,7 +57,7 @@ tests=( $(find_tests ${1:-.*}) )
 
 # test-cmd specific defaults
 TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-cmd}"
+export BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-cmd}"
```

Comment:
I see.. you're creating files in `test/cmd/newapp.sh` and putting them in the temp dir... I think that's a bad idea.
How about writing a `checkResult` like below?
https://github.com/openshift/origin/pull/6615/files#diff-d47c059acde3ba8455be65d7c690ac25R1132